test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
Okay, as long as it's no trouble. I'll dedicate some time to that task as well as cleaning up this PR.